pkg/webhooks/webhookpatch.go

```diff
@@ -30,13 +30,16 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	"istio.io/api/label"
+	"istio.io/istio/pilot/pkg/keycertbundle"
 	"istio.io/istio/pkg/queue"
+	"istio.io/istio/pkg/webhooks/util"
 	"istio.io/pkg/log"
 )
 
 var (
 	errWrongRevision     = errors.New("webhook does not belong to target revision")
 	errNoWebhookWithName = errors.New("webhook configuration did not contain webhook with target name")
+	whc                  *cache.ListWatch
```

Comment:
We cannot use global variables, can you keep it scoped in functions/WebhookCertPatcher?

Reply:
sure, fixed